paper-icon-button.html

```diff
@@ -135,6 +139,21 @@
        */
       icon: {
         type: String
+      },
+
+      /**
+       * Specifies the alternate text for the button, for accessibility.
+       */
+      alt: {
+        type: String,
+        observer: "_altChanged"
+      }
+    },
+
+    _altChanged: function() {
+      var label = this.getAttribute('aria-label');
+      if (!label) {
```

Comment:
Oh hmm. I had written this to make sure that `aria-label` wins over `alt`. I'm not sure how we can do both :(